tests/test_interface.rb

```diff
@@ -778,6 +779,29 @@ def test_interface_ipv4_addr_mask_set_netmask_invalid
     interface_ethernet_default(interfaces_id[0])
   end
 
+  def test_create_acl
```

Comment:
You'll need to rewrite these tests. Here's a simple example, just repeat it for ipv6:

```
  def test_ipv4_acl
    # Sample cli:
    #
    #   interface Ethernet1/1
    #     ip access-group foo in
    #     ip access-group bar out
    #
    interface_ethernet_default(interfaces[0])
    i = Interface.new(interfaces[0])

    i.ipv4_acl_in = 'foo'
    assert_equal('foo', i.ipv4_acl_in)
    i.ipv4_acl_out = 'bar'
    assert_equal('bar', i.ipv4_acl_out)

    i.ipv4_acl_in = 'foo2'
    assert_equal('foo2', i.ipv4_acl_in)
    i.ipv4_acl_out = 'bar2'
    assert_equal('bar2', i.ipv4_acl_out)

    i.ipv4_acl_in = i.default_ipv4_acl_in
    assert_equal('foo2', i.ipv4_acl_in)
    i.ipv4_acl_out = i.default_ipv4_acl_out
    assert_equal('bar2', i.ipv4_acl_out)
  end
```